client/app/components/atoms/Button/__tests__/Button.test.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import renderer from 'react-test-renderer';
+// import { createShallow } from '@material-ui/core/test-utils';
```

Comment:
```suggestion
// import { createShallow } from '@material-ui/core/test-utils';
```
Comment out code should be removed if not needed.